queries.sql

```diff
@@ -8,5 +8,52 @@ SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT * FROM animals WHERE name NOT IN('Gabumon');
 
+/*transaction queries*/
+BEGIN; 
+UPDATE animals SET species='unspecified';
+SELECT * as animals FROM animals;
+ROLLBACK;
 
+BEGIN;
+UPDATE animals
+SET species='digimon'
+WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon'
+WHERE species IS NULL;
+SELECT * as animals FROM animals;
+COMMIT;
 
+BEGIN;
+DELETE FROM ANIMALS;
+SELECT * as animals FROM animals;
+ROLLBACK;
+
```

Comment:
- [ ] Here also, you were asked to check the before and after the rollback, here you have only checked for the before